src/components/login/oauth-button-group.js

```diff
@@ -0,0 +1,28 @@
+import { Button, ButtonGroup, VisuallyHidden } from '@chakra-ui/react'
+
+import { GitHubIcon, GoogleIcon, TwitterIcon } from './provider-icons'
+const providers = [
+  {
+    name: 'Google',
+    icon: <GoogleIcon boxSize='5' />,
+  },
+  {
+    name: 'Twitter',
+    icon: <TwitterIcon boxSize='5' />,
+  },
+  {
+    name: 'GitHub',
+    icon: <GitHubIcon boxSize='5' />,
+  },
+]
```

Comment:
Wouldn't it be better to avoid having extra file `provider-icons.js` just for icons, since we have already icon package.

```suggestion
const providers = [
  {
    name: 'Google',
    icon: <Box as={FaGoogle} color='red.500' boxSize='5' />,
  },
  {
    name: 'Twitter',
    icon: <Box as={FaTwitter} color='twitter.500' boxSize='5' />,
  },
  {
    name: 'Facebook',
    icon: <Box as={FaFacebook} color='facebook.500' boxSize='5' />,
  },
]
```